props/saber_fett263_buttons.h

```diff
@@ -3616,6 +3852,79 @@ SaberFett263Buttons() : PropBase() {}
     SelectSFXEvenOdd(&SFX_preon);
   }
 
+  void ToggleBattleMode() {
+    if (!battle_mode_) {
+      battle_mode_ = true;
+      if (SFX_bmbegin) {
+        hybrid_font.PlayCommon(&SFX_bmbegin);
+      } else {
+        hybrid_font.DoEffect(EFFECT_FORCE, 0);
+      }
+    } else {
+      battle_mode_ = false;
+      if (SFX_bmend) {
+        hybrid_font.PlayCommon(&SFX_bmend);
+      } else {
+        beeper.Beep(0.5, 3000);
+      }
+    }	  
+  }
+
+  void ToggleMultiBlast() {
+    swing_blast_ = !swing_blast_;
+    if (swing_blast_) {
+      if (SFX_blstbgn) {
+        hybrid_font.PlayCommon(&SFX_blstbgn);
+      } else {
+        hybrid_font.SB_Effect(EFFECT_BLAST, 0);
+      }
+    } else {
+      if (SFX_blstend) {
+        hybrid_font.PlayCommon(&SFX_blstend);
+      } else {
+        hybrid_font.SB_Effect(EFFECT_BLAST, 0);
+      }
+    }
+  }
+
+  void ToggleBattleModeMultiBlast() {
+    if (battle_mode_ && !swing_blast_) {
+      check_blast_ = true;
+      last_blast_millis_ = millis();
+    }
+    swing_blast_ = false;
+  }
+
+  void DoCheckMultiBlast() {
```

Comment:
Another CheckSOMETHING function that doesn't return a bool.
Either needs a new name or a comment.

Reply:
renamed to DoAutoMultiBlast()